manifest.json

```diff
@@ -1,52 +1,55 @@
 {
-  "name": "Pixelfed",
-  "id": "pixelfed",
-  "packaging_format": 1,
-  "description": {
-    "en": "Federated Image Sharing"
-  },
-  "version": "0.0.0~ynh2",
-  "url": "https://pixelfed.org/",
-  "license": "MIT",
-  "requirements": {
-		"yunohost": ">= 3.0.0"
-  },
-  "maintainer": {
-    "name": "jibec",
-    "email": "jean-baptiste@holcroft.fr"
-  },
-  "multi_instance": false,
-  "services": [
-    "nginx",
-    "php7-fpm"
-  ],
-  "arguments": {
-    "install": [
-      {
-        "name": "domain",
-        "ask": {
-          "en": "Choose a domain for Pixelfed"
-        },
-        "example": "domain.org",
-        "type": "domain"
-      },
-      {
-        "name": "path",
-        "ask": {
-          "en": "Choose a path for Pixelfed"
-        },
-        "type": "path",
-        "example": "/pixelfed",
-        "default": "/pixelfed"
-      },
-      {
-        "name": "is_public",
-        "ask": {
-          "en": "Is it a public site ?"
-        },
-        "type": "boolean",
-        "default": "0"
-      }
-    ]
-  }
+    "name": "Pixelfed",
+    "id": "pixelfed",
+    "packaging_format": 1,
+    "description": {
+        "en": "ActivityPub Federated Image Sharing",
+        "fr": "Partage d'image fédéré avec ActivityPub."
+    },
+    "version": "0.0.0~ynh2",
```

Comment:
please change the package version, at least the ynh version

Reply:
Change made. I didn't changed it because it was just code reorganisation